tools/lab_dispatch/resources/input.conf_template

```diff
@@ -0,0 +1,40 @@
+[launcher]
+launcher_version = 1
+stock_reports = certification, text
+
+[test plan]
+unit = $INPUT_TEST_PLAN
+forced = yes
+
+[test selection]
+forced = true
+match=$INPUT_MATCH
+
+[ui]
+type = silent
+
+[environment]
```

Comment:
I have a couple of suggestions here:
- In the jobs I am aware of, the `[environment]` section is split into a separate file (which is _annoyingly_ named `checkbox.conf`). And, at some point in the near future, you may be receiving that separate environment file from an endpoint or a different source (i.e. not storing it in your repo) because there needs to be a single source of truth. So, in the interest of consistency with other jobs _and_ perhaps seeking to minimise future changes, it might be a good idea to split this file into the "head" and the environment section.
- Continuing from the previous suggestion, I am not too crazy about the name `input.conf_template`. I understand why you named it that way but I don't think it reflects the contents of the file. So, if you agree with splitting the file, my suggestions would be:
  - `head.template.conf` (later turning into `head.conf` after you've instantiated the input)
  - `environment.template.conf (later turning into `environment.partial.conf` after you've instantiated the input and `environment.conf` after the next instantiation on the agent)

Reply:
I think this is more complicated (as I need to do 2 substitutions or stack first and then substitute) and I don't see a benefit with it. Once we have an environment endpoint, we can just remove the envvars from the file and shift it to a fetch like we do with the manifests. I like the fact that the conf file is relatively small and readable, one can open it and modify it (for example to get some temporary testing done) with relative ease. I don't want to be in the situation where you are left wondering where are the values set and why.

Reply:
Would you consider renaming this single configuration file to something different? Right now, the naming trajectory is:

`input.conf_template` --> (partially instantiated in the runner)
`input.conf` --> (fully instantiated in the agent)
`test_conf.conf` --> (stacked with manifest)
`checkbox.conf`

_One_ suggestion would be:

`checkbox.no-manifest.template.conf` --> (partially instantiated in the runner)
`checkbox.no-manifest.partial.conf` --> (fully instantiated in the agent)
`checkbox.no-manifest.conf` --> (stacked with manifest)
`checkbox.conf`

The naming decisions for these files are supposed to indicate what each configuration fragment contains. And so far the names we have selected in our SRU and snap-testing jobs fall very short of that. Anyway, again up to you.

Reply:
Good idea, done